lib/sycamore/sycamore/tests/unit/query/test_plan.py

```diff
@@ -212,5 +212,5 @@ def test_compare_plans_structure_changed(llm_filter_plan):
     assert diff[0].diff_type == LogicalNodeDiffType.PLAN_STRUCTURE
     assert isinstance(diff[0].node_a, LlmFilter)
     assert isinstance(diff[0].node_b, LlmFilter)
-    assert len(diff[0].node_a.downstream_nodes) == 1
-    assert len(diff[0].node_b.downstream_nodes) == 0
+    assert len(diff[0].node_a.get_downstream_nodes()) == 1
```

Comment:
Not sure if this is the right place, but do we want to test that the values of `get_downstream_nodes()`/`get_dependencies()` are what we expect, and also that the `.dependencies` and `.downstream_nodes` properties return what we expect?

Reply:
Let's discuss offline, I hadn't done this initially because I didn't want to make assumptions on ids being stable. But we sort of already do that in other places so maybe it doesn't matter

Reply:
I added a node_id assertion